jbpm-human-task/jbpm-human-task-jpa/src/main/java/org/jbpm/services/task/persistence/JPATaskPersistenceContext.java

```diff
@@ -128,20 +129,27 @@ public Task persistTask(Task task) {
         	this.em.flush();
             return this.em.find(TaskImpl.class, task.getId(), LockModeType.PESSIMISTIC_FORCE_INCREMENT );
```

Comment:
Now I see this: Shouldn't `create()` be called if `pessimisticLocking` is enabled too?

Reply:
It seems to be you are right. `create()` should be called before the if.